client/src/components/Admin/Issue.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Button from '../Button';
+import Card from '../Card';
 import IconText from '../IconText';
 import Pin from './Pin';
 import { adminCloseIssue } from '../../actionCreators/adminButtons';
 import { getIssue } from '../../selectors/issues';
 import '../../css/IssueAdmin.css';
 
 const Issue = ({ closeIssue, issue }) => (
-  <div className="IssueAdmin">
-    <div className="IssueAdmin-heading">
+  <Card
+    classes="IssueAdmin"
+  >
+    <div>
       <Pin code="DEADBEEF" />
-      <h2 className="IssueAdmin-title">Aktiv sak</h2>
+      <p className="IssueAdmin-title">Aktiv sak</p>
```

Comment:
That one was sort of a pickle. I couldn't elegantly fit the pin and "title" into the card subtitle, so I just let them be card content. Going off of the style that was there previously it didn't seem to be styled particularly prominently, so I mostly let it be. h2 is sort of big regardless, h4 could be used here potentially.